edgelet/edgelet-kube/src/runtime.rs

```diff
@@ -237,6 +237,7 @@ where
                             os_type: "Kubernetes".to_string(),
                             architecture: serde_json::to_string(&architectures).unwrap(),
                             version: edgelet_core::version_with_source_version(),
+                            provisioning_type: not_supported.clone(),
```

Comment:
I need to find out about the expected behavior on k8s.

Reply:
We support the same as single node. See https://github.com/Azure/iotedge/blob/master/kubernetes/charts/edge-kubernetes/values.yaml

Reply:
Because our k8s support is on a different release timeline and therefore not part of the 1.0.10 release, I'd like to add k8s support as a separate PR.

Reply:
Sounds good. K8s is not needed for now. We should confirm with Venkat.

Reply:
@veyalla?

Reply:
Agree with Cindy, k8s not needed for now.